README

```diff
@@ -1,14 +1,14 @@
-WiredTiger 3.2.2: (August 28, 2019)
+WiredTiger 3.3.0: (March 20, 2020)
```

Comment:
The other PR went to version 10.0. Is 3.3.0 the right version for this? I admit I've lost track of what PR is exactly doing what as there has been several of them and they're nuanced.